id_ID.ini

```diff
@@ -404,24 +404,24 @@ Autoconfigure = Konfigurasi otomatis
 Autoconfigure for device = Konfigurasi otomatis utk perangkat
 Clear All = Bersihkan Semua
 Default All = Default-kan Semua
-Map a new key for = Petakan tombol baru untuk
-Test Analogs = Coba Analog
+Map a new key for = Petakan untuk tombol baru
+Test Analogs = Uji Analog
 
 [MainMenu]
 Browse = Telusuri...
 Credits = Penghargaan
-DownloadFromStore = Unduh dari PPSSPP Homebrew Store
+DownloadFromStore = Unduh dari PPSSPP Toko Rumahbrew
 Exit = Keluar
 Game Settings = Pengaturan permainan
-Games = Permainan
+Games = Cari Permainan
```

Comment:
It seems that "Cari Permainan" means "Search Games".
I don't see why you would do this change, I think "Games = Permainan" is more appropriate.
Nope ?